go.mod

```diff
@@ -13,42 +13,42 @@ require (
-	k8s.io/kubelet v0.20.4
-	k8s.io/kubernetes v1.20.4
+	k8s.io/kubelet v0.20.5
+	k8s.io/kubernetes v1.20.5
 	k8s.io/utils v0.0.0-20210111153108-fddb29f9d009
 	sigs.k8s.io/controller-runtime v0.8.1
 )
 
 replace (
-	k8s.io/api => k8s.io/api v0.20.4
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.20.4
-	k8s.io/apimachinery => k8s.io/apimachinery v0.21.0-alpha.0
-	k8s.io/apiserver => k8s.io/apiserver v0.20.4
-	k8s.io/cli-runtime => k8s.io/cli-runtime v0.20.4
-	k8s.io/client-go => k8s.io/client-go v0.20.4
-	k8s.io/cloud-provider => k8s.io/cloud-provider v0.20.4
-	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.20.4
-	k8s.io/code-generator => k8s.io/code-generator v0.20.5-rc.0
-	k8s.io/component-base => k8s.io/component-base v0.20.4
-	k8s.io/component-helpers => k8s.io/component-helpers v0.20.4
-	k8s.io/controller-manager => k8s.io/controller-manager v0.20.4
-	k8s.io/cri-api => k8s.io/cri-api v0.20.5-rc.0
-	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.20.4
-	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.20.4
-	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.20.4
-	k8s.io/kube-proxy => k8s.io/kube-proxy v0.20.4
-	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.20.4
-	k8s.io/kubectl => k8s.io/kubectl v0.20.4
-	k8s.io/kubelet => k8s.io/kubelet v0.20.4
-	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.20.4
-	k8s.io/metrics => k8s.io/metrics v0.20.4
-	k8s.io/mount-utils => k8s.io/mount-utils v0.20.5-rc.0
-	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.20.4
-	k8s.io/sample-cli-plugin => k8s.io/sample-cli-plugin v0.20.4
-	k8s.io/sample-controller => k8s.io/sample-controller v0.20.4
+	k8s.io/api => k8s.io/api v0.20.5
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.20.5
+	k8s.io/apimachinery => k8s.io/apimachinery v0.20.6-rc.0
+	k8s.io/apiserver => k8s.io/apiserver v0.20.5
+	k8s.io/cli-runtime => k8s.io/cli-runtime v0.20.5
+	k8s.io/client-go => k8s.io/client-go v0.20.5
+	k8s.io/cloud-provider => k8s.io/cloud-provider v0.20.5
+	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.20.5
+	k8s.io/code-generator => k8s.io/code-generator v0.20.6-rc.0
+	k8s.io/component-base => k8s.io/component-base v0.20.5
+	k8s.io/component-helpers => k8s.io/component-helpers v0.20.5
+	k8s.io/controller-manager => k8s.io/controller-manager v0.20.5
+	k8s.io/cri-api => k8s.io/cri-api v0.20.6-rc.0
+	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.20.5
+	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.20.5
+	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.20.5
+	k8s.io/kube-proxy => k8s.io/kube-proxy v0.20.5
+	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.20.5
+	k8s.io/kubectl => k8s.io/kubectl v0.20.5
+	k8s.io/kubelet => k8s.io/kubelet v0.20.5
+	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.20.5
+	k8s.io/metrics => k8s.io/metrics v0.20.5
+	k8s.io/mount-utils => k8s.io/mount-utils v0.20.6-rc.0
```

Comment:
This one is also 0.20.6-rc.0. Can you explain why it got here?

Reply:
@bart0sh @kad This is the reason why we get those:
```go mod download -json k8s.io/apimachinery@kubernetes-1.20.5
{
	"Path": "k8s.io/apimachinery",
	"Version": "v0.20.6-rc.0",
	...
}```

the script picks up that `Version`